apollo-router/src/state_machine.rs

```diff
@@ -342,17 +327,9 @@ where
             )
             .await
         {
-            Ok((new_router_service, plugins)) => {
-                let plugin_handlers: HashMap<String, Handler> = plugins
-                    .iter()
-                    .filter_map(|(plugin_name, plugin)| {
-                        (plugin_name.starts_with("apollo.")
-                            || plugin_name.starts_with("experimental."))
-                        .then(|| plugin.custom_endpoint())
-                        .flatten()
-                        .map(|handler| (plugin_name.clone(), handler))
-                    })
-                    .collect();
+            Ok(new_router_service) => {
+                let plugin_handlers =
+                    new_router_service.custom_endpoints();
```

Comment:
Looks like `cargo fmt` wants this in CI.

```suggestion
                let plugin_handlers = new_router_service.custom_endpoints();
```